eng/SnippetGenerator/CSharpProcessor.cs

```diff
@@ -11,8 +11,8 @@ namespace SnippetGenerator
 {
     public class CSharpProcessor
     {
-        private static readonly string _snippetFormat = "{3} <code snippet=\"{0}\">{1}{2} </code>";
-        private static readonly string _snippetExampleFormat = "{3} <example snippet=\"{0}\">{1}{3} <code>{1}{2} </code>{1}{3} </example>";
+        private static readonly string _snippetFormat = "{3} <code snippet=\"{0}\" language=\"csharp\">{1}{2} </code>";
+        private static readonly string _snippetExampleFormat = "{3} <example snippet=\"{0}\">{1}{3} <code language=\"csharp\">{1}{2} </code>{1}{3} </example>";
 
         private static readonly Regex _snippetRegex = new Regex("^(?<indent>\\s*)\\/{3}\\s*<code snippet=\"(?<name>[\\w:]+)\">.*?\\s*<\\/code>",
```

Comment:
would this regex still match?

Reply:
The tests pass and global regen worked.

Reply:
Those things might be true, but the regex won't match the new format.

Reply:
Those things might be true, but the regex won't match the new format.

Reply:
Let me re-validate. The tests run the generator twice to ensure we re-process successfully, but I may be missing something.

Reply:
You were right 😄